sdk/attestation/Azure.Security.Attestation/src/AttestationToken.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Azure.Core;
+using Azure.Security.Attestation.Models;
+using System.Security.Cryptography.X509Certificates;
+using Azure.Security.Attestation.Common;
+using System.Text;
+
+namespace Azure.Security.Attestation
+{
+    /// <summary>
+    /// Represents an Attestation Token object.
+    /// </summary>
+    public class AttestationToken
+    {
+        protected private string _token;
+        private JsonSerializerOptions _options = new JsonSerializerOptions();
```

Comment:
nit: why not just initialize this in the ctor?

Reply:
I have always felt that it was better to initialize fields at the declaration if the initialization didn't depend on constructor parameters. Is that not correct?